client/trino-jdbc/src/test/java/io/trino/jdbc/TestTrinoDriver.java

```diff
@@ -1144,4 +1163,26 @@ private static Properties toProperties(Map<String, String> map)
         map.forEach(properties::setProperty);
         return properties;
     }
+
+    public static class TestDnsResolver
```

Comment:
Name this `TestingDnsResolver` since the `Test` prefix is reserved for classes that have tests, whereas this is an implementation for something "for testing"